eng/versioning/version_client.txt

```diff
@@ -101,17 +101,17 @@ com.azure:azure-core-test;1.27.0-beta.9;1.27.0-beta.10
 com.azure:azure-core-tracing-opentelemetry;1.0.0-beta.57;1.0.0-beta.58
 com.azure:azure-core-tracing-opentelemetry-samples;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-core-version-tests;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-cosmos;4.71.0;4.72.0-beta.1
+com.azure:azure-cosmos;4.71.0;4.72.0
 com.azure:azure-cosmos-benchmark;4.0.1-beta.1;4.0.1-beta.1
 com.azure.cosmos.spark:azure-cosmos-spark_3_2-12;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-cosmos-encryption;2.20.0;2.21.0-beta.1
+com.azure:azure-cosmos-encryption;2.20.0;2.21.0
 com.azure.cosmos.spark:azure-cosmos-spark_3-3_2-12;4.37.2;4.38.0-beta.1
 com.azure.cosmos.spark:azure-cosmos-spark_3-4_2-12;4.37.2;4.38.0-beta.1
 com.azure.cosmos.spark:azure-cosmos-spark_3-5_2-12;4.37.2;4.38.0-beta.1
 com.azure.cosmos.spark:azure-cosmos-spark-account-data-resolver-sample;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-cosmos-test;1.0.0-beta.12;1.0.0-beta.13
 com.azure:azure-cosmos-tests;1.0.0-beta.1;1.0.0-beta.1
-com.azure.cosmos.kafka:azure-cosmos-kafka-connect;2.3.1;2.4.0-beta.1
+com.azure.cosmos.kafka:azure-cosmos-kafka-connect;2.3.1;2.4.0
```

Comment:
@xinlian12 do we need 2.4.0 - I would assume 2.3.2 is sufficient, because this is only a bugfix - and since all depdnencies are shaded no need to increase minor version number. Unleass you consider intorducing teh cache a new fetaure and worth bumping minor verison number?

Reply:
yea Fabian, I took this change as a overall optimization, so minor version bump is okie I think in this case